.ci/scripts/prepare_slack_data.py

```diff
@@ -0,0 +1,189 @@
+                            },
+                        },
+                        {
+                            "type": "divider",
+                        },
+                        {
+                            "type": "section",
+                            "text": {
+                                "type": "mrkdwn",
+                                "text": message,
+                            },
+                        },
+                        {
+                            "type": "divider",
+                        },
+                        {
+                            "type": "actions",
+                            "elements": [
+                                {
+                                    "type": "button",
+                                    "text": {
+                                        "type": "plain_text",
+                                        "text": "kibana link",
+                                    },
+                                    "style": "primary",
+                                    "url": f"{kibana_url}",
+                                    "action_id": "kibana-instance-button",
+                                },
+                                {
+                                    "type": "button",
+                                    "text": {
+                                        "type": "plain_text",
+                                        "text": "state bucket",
+                                    },
+                                    "style": "primary",
+                                    "url": f"{s3_bucket}",
+                                    "action_id": "s3-bucket-button",
+                                },
+                                {
+                                    "type": "button",
+                                    "text": {
+                                        "type": "plain_text",
+                                        "text": "action run",
+                                    },
+                                    "style": "primary",
+                                    "url": f"{github_run_url}",
+                                    "action_id": "action-run-button",
+                                },
+                                {
+                                    "type": "button",
+                                    "text": {
+                                        "type": "plain_text",
+                                        "text": "docs",
+                                    },
+                                    "style": "primary",
+                                    "url": docs_url,
+                                    "action_id": "docs-button",
+                                },
+                            ],
+                        },
+                    ],
+                },
+            ],
+        }
+        set_output("payload", json.dumps(slack_payload))
+
+    except BuildSlackException as err:
```

Comment:
could other exceptions occur? (e.g., issues with network, JSON operations). Consider adding a generic exception handler or specific handlers for known exception types that could arise. What is going to throw `BuildSlackException`?

Reply:
As mentioned earlier, `BuildSlackException` is an empty wrapper. If other exceptions occur, it will be raised as a general exception. If issues related to JSON operations arise, the corresponding JSON exception will be raised. Since this script is basic, providing specific handlers or a generic exception handler is not necessary in this case.